src/main/java/com/techsophy/tsf/account/constants/PropertyConstant.java

```diff
@@ -0,0 +1,10 @@
+package com.techsophy.tsf.account.constants;
+
+public class PropertyConstant {
+    public static final String ADD_USER_ENDPOINT = "http://localhost:8084/internal/v1/users/create";
```

Comment:
what is this endpoint for? That too a constant. Cant think of any reason why such a thing might be required